src/lib_ccx/ccx_encoders_helpers.c

```diff
@@ -31,16 +44,68 @@ static const char *spell_builtin[] =
 	NULL
 };
 
+static const char *profane_builtin[] =
+{
+	"arse",
+	"ass",
```

Comment:
what if someone means the donkey? 🤔

Reply:
I don't know I just copy and pasted the words from https://en.wikipedia.org/wiki/Category:Profanity and removed the ones that could totally be used, in a non profane way.

Also I didn't know they meant donkey

Reply:
That's fine, we can leave this list :-) If someone wants a different list they can just edit it.